docs/debug.md

```diff
@@ -0,0 +1,21 @@
+# Troubleshooting
+The following is a (growing) list of possible errors you might run in to when setting up the CSI Driver provider for auzre, along with recommendations for remediation.
```

Comment:
nit: add new line

Reply:
Below the Troubleshooting header or do you mean another below the intro?